Source/Layout/ASStackLayoutDefines.h

```diff
@@ -15,6 +15,10 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutDirection) {
   ASStackLayoutDirectionVertical,
   /** Children are stacked horizontally */
   ASStackLayoutDirectionHorizontal,
+  /** Children are stacked vertically, but in reverse. Only used by Yoga spec. */
```

Comment:
We may should compile this part out for now via the `Yoga` flag. Same in `ASStackLayoutSpec.mm` where we use the flags.

Reply:
You mean where we just assert on them? lol